website/source/docs/guides/creating-certificates.html.md

```diff
@@ -32,140 +32,141 @@ signed by this CA will be allowed to communicate with the cluster.
-$ echo '{"key":{"algo":"rsa","size":2048}}' | cfssl gencert -ca=consul-ca.pem -ca-key=consul-ca-key.pem -config=cfssl.json \
-    -hostname="client.global.consul,localhost,127.0.0.1" - | cfssljson -bare client
+If you enforce HTTPS you will need a certificate in order to use consul commands
+or curl to access the HTTPS API.
 
-# Generate a certificate for the CLI
-$ echo '{"key":{"algo":"rsa","size":2048}}' | cfssl gencert -ca=consul-ca.pem -ca-key=consul-ca-key.pem -profile=client \
-    - | cfssljson -bare cli
+Create a CLI certificate:
+
+```shell
+$ consul tls cert create -cli
+==> Using consul-ca.pem and consul-ca-key.pem
+==> Saved consul-cli-0.pem
+==> Saved consul-cli-0-key.pem
 ```
 
+### Note on SANs for Server and Client Certificates
+
 Using `localhost` and `127.0.0.1` as subject alternate names (SANs) allows
 tools like `curl` to be able to communicate with Consul's HTTP API when run on
 the same host. Other SANs may be added including a DNS resolvable hostname to
 allow remote HTTP requests from third party tools.
 
+### What goes where
+
 You should now have the following files:
 
-* `cfssl.json` - cfssl configuration.
-* `consul-ca.csr` - CA signing request.
-* `consul-ca-key.pem` - CA private key. Keep safe!
 * `consul-ca.pem` - CA public certificate.
-* `cli.csr` - Consul CLI certificate signing request.
-* `cli-key.pem` - Consul CLI private key.
-* `cli.pem` - Consul CLI certificate.
-* `client.csr` - Consul client node certificate signing request for the `global` region.
-* `client-key.pem` - Consul client node private key for the `global` region.
-* `client.pem` - Consul client node public certificate for the `global` region.
-* `server.csr` - Consul server node certificate signing request for the `global` region.
-* `server-key.pem` - Consul server node private key for the `global` region.
-* `server.pem` - Consul server node public certificate for the `global` region.
-
-Each Consul node should have the appropriate key (`-key.pem`) and certificate
-(`.pem`) file for its region and role. In addition each node needs the CA's
-public certificate (`consul-ca.pem`).
-
-Please note you will need the keys for the CLI if you choose to disable
-HTTP (in which case running the command `consul members` will return an error).
-This is because the Consul CLI defaults to communicating via HTTP instead of
-HTTPS. We can configure the local Consul client to connect using TLS and specify
-our custom keys and certificates using the command line:
+* `consul-ca-key.pem` - CA private key. Keep safe!
+* `consul-server-dc1-0.pem` - Consul server node public certificate for the `dc1` datacenter.
+* `consul-server-dc1-0-key.pem` - Consul server node private key for the `dc1` datacenter.
+* `consul-client-0.pem` - Consul client node public certificate.
+* `consul-client-0-key.pem` - Consul client node private key.
+* `consul-cli-0.pem` - Consul CLI certificate.
+* `consul-cli-0-key.pem` - Consul CLI private key.
+
+Here is a config for a server:
```

Comment:
Could we be a little more specific: “Here is an example agent configuration for Consul servers”.

Reply:
Did exactly that! 👍